### PR TITLE
Add 'Call ringtone' setting to change ringtone for incoming calls.

### PR DESCRIPTION
**Pull request by juanitobananas**
Okay, second try. The comment from my first PR (#826) still applies (well, most of it):

Add 'Notification Sounds' category in settings with a 'Call
ringtone' setting and a standard ringtone picker to change it.

I am not sure if you even want to have something like this. I think it is pretty cool :wink: (and pretty common). IMHO the current ring.ogg is a little annoying. I haven't (yet?) deleted the original code.

I hope you like it. Feel free to ignore this if you don't :smile:

I am not sure if creating a RingtoneUtils was a good idea, as part of the code is still VectorCallSoundManager but I find it easier to handle like this.

I will gladly change some parts/style if you want me too. Just say so!

Cheers

### CODE REVIEW

**Comment by dva-re** [comment]
pls, don`t quit this idea.

File: vector/src/main/res/xml/vector_settings_preferences.xml [hunk 1]
```diff
@@ -77,6 +77,16 @@
 
     <im.vector.preference.VectorDividerCategory />
 
+    <PreferenceCategory android:title="@string/notification_sounds_settings">
+
+        <im.vector.preference.VectorCustomActionEditTextPreference
+            android:key="@string/notification_sounds_settings_ringtone"
```
Comment (bmarty) [inline vector/src/main/res/xml/vector_settings_preferences.xml]: key cannot be localized string (it has been fixed for the other preferences)

**Comment by juanitobananas** [comment]
Hi @bmarty,

No problem, thanks for the (late) response! Of course, I'll try to rebase it. I am currently playing around with kotlin, so that might just work :wink: 

I don't think I'll find time for this until September though. But I'd be glad to do it. If you happen to be in a hurry, I am afraid you'll have to do it yourself or find somebody else.

Cheers,
Juanito

**Comment by bmarty** [comment]
Hi @juanitobananas ,

Ok, there is no rush :)

Benoît

**Comment by bmarty** [comment]
Replaced by #2730

**Comment by bmarty** [comment]
Hello @juanitobananas ,
No problem, we apologize for a such long delay to deal with your interesting PR :)
Benoit

**Comment by sl5net** [comment]
ring.ogg  is still ringing at my android. i tried change all sound but ists still rining. and i deleted ring.ogg from the file system (dosent help)

**Pull request closed without merging**